features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/telemetry/internal/TelemetryEventHandler.kt

```diff
@@ -440,16 +440,17 @@ internal class TelemetryEventHandler(
         // Would be nice to add the test with the flavor which is has no io.opentracing and test
         // for obfuscation enabled case.
         return try {
-            val globalTracerClass = Class.forName("io.opentracing.util.GlobalTracer")
+            val globalDatadogTracerHolderClass =
```

Comment:
I don't see this GlobalDatadogTracerHolder class anywhere in the project - was it renamed?